PlayerCore/components/New VRM Core/VRMErroredItems.swift

```diff
@@ -0,0 +1,51 @@
+//  Copyright 2018, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+
+public struct VRMErroredItems {
+    static let initial = VRMErroredItems(fetching: [],
+                                         parsing: [],
+                                         timeout: [],
+                                         redirect: [])
+    
+    public let fetching: Set<VRMCore.Item>
```

Comment:
I would rather split this into 4 components - currently it will be easy to make an error in the reducer.